src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1580,16 +1570,12 @@ protected override void ProcessRecord()
         internal long SetRequestContent(HttpRequestMessage request, byte[] content)
         {
             ArgumentNullException.ThrowIfNull(request);
+            ArgumentNullException.ThrowIfNull(content);
 
-            if (content is null)
-            {
-                return 0;
-            }
-
-            var byteArrayContent = new ByteArrayContent(content);
+            ByteArrayContent byteArrayContent = new(content);
             request.Content = byteArrayContent;
 
-            return byteArrayContent.Headers.ContentLength.Value;
+            return 0;
```

Comment:
We can make all these `SetRequestContent` methods `void`.

Reply:
I don't think we can `SetRequestContent(HttpRequestMessage request, IDictionary content)` returns `SetRequestContent`

Reply:
I don't understand your comment. Before the PR all `SetRequestContent` returned `long`. If we don't use the result we can convert these methods to `void`.

Reply:
I thought I had to `return SetRequestContent(request, body);`, instead I could simply `SetRequestContent(request, body);`.
`SetRequestContent` is now void